README.md

```diff
@@ -351,51 +387,76 @@ public function getDynamicSEOData(): SEOData
 {
     return new SEOData(
         // ...
-        schema: SchemaCollection::initialize()->addArticle(),
+        schema: SchemaCollection::make()->withArticle(),
```

Comment:
Do you think we should change the naming of the methods? I'm OK if it provides a functional benefit, but if not I would prefer to keep it the same.